oidc-authentication.md

```diff
@@ -241,10 +241,119 @@ As defined by [OIDC]:
+
+Authorization systems sometimes require the ability to perform access control
+decisions _without_ authentication, without identifying who is performing an
+action, either because the protected resource is sufficiently low-value (or
+low-threat if stolen), or due to practical requirements such as when sharing an
+online document for dissemination and collaboration with an unknown audience.
+
+As a result, most deployed authorization systems tend to employ a _hybrid_
+paradigm: they allow both identity-based and capability-based access control.
+Systems that prefer strict identity based Access Control Lists nevertheless
+develop capability links (unguessable tokens or URLs that allow access to the
+resource). Similarly, even systems such as [Authorization Capabilities]()
+(originally known as Object Capabilities) that are staunchly _against_
+authentication or using identifiers for access control, are forced to
+accommodate authentication requirements (if only for purposes of auditing) --
+see, for example, the [Horton protocol](http://www.erights.org/download/horton/document.pdf)
+used by the object capabilities community.
+
+Although authorization is largely out of scope for this document, it is worth
+mentioning that this specification aims to support this hybrid approach, and
+provide functionality for authentication as well as token-based capabilities.
 
 ## 2.2 Identity
 
+For a given [entity]() (whether a person, an organization or a software agent),
+the term [identity]() refers to a combination of: an [identifier]() (such as a
+WebID or a DID), and a set of [claims]() (metadata and other attributes) that
+typically reside in an [identifier document]() (such as a WebID Profile or a
+DID Document).
+
+In this specification, we are concerned primarily with two types of identities.
+
+The first one is the identity of users (or [controller]()s), which are entities
+responsible for an action performed on a protected resource.
+
+The second one is the [client]() identity, since the WAC authorization system
+often requires the knowledge of what software client was used to perform an
+action, and restricts access accordingly.
+
+Whenever possible, this specification prefers identifiers that are _resolvable_,
+and result in identifier documents.
+
+### 2.2.1 Controller Identifiers (WebID / DID)
+
+- For people and organizations only. For application or service identity, see
+  sections 2.2.3 and 2.2.4.
+- Link to the WebID section of the spec.
+- Globally unique, non-pairwise-pseudonymous.
+
+### 2.2.2 Classifying Clients
+
+There are many different kinds of software that we’re including in the broad
+term [client]().
+
+**We can classify clients by:**
+
+- By deployment type: Server-side web apps, in-browser JS web apps, mobile
+  apps, desktop apps, constrained apps and IoT devices.
+- Clients that have their own identity (bots/services) vs clients that are
+  always “piloted” by a user (apps)
+- “Attended” vs “Unattended” clients
```

Comment:
Could you provide a description of what his is? Is this not a repeat of the bullet above?

Reply:
Sure thing